Предположим, что в некоторой популяции до 60 лет доживает 50%, а до 80 лет — 20%. Какова вероятность (от 0 до 1), что случайно выбранный шестидесятилетний представитель популяции доживёт до восьмидесяти? Запишите ответ с точностью до одного знака после десятичной точки.
 
Используйте определение условной вероятности.
P A|B =P(AB)/P(B)

In [77]:
let60 = 0.5
let80 = 0.2

let80/let60

0.4

Какова вероятность того, что плазму одного случайно выбранного австралийца можно перелить другому случайно выбранному австралийцу? 

Искомая вероятность — сумма вероятностей всех ячеек, отмеченных плюсом. Поскольку австралийцы выбираются независимо, в каждой ячейке вероятность равна произведению вероятностей соответствующих строки и столбца.

In [78]:
import numpy as np

p1=0.49
p2=0.38
p3=0.1
p4=0.03

p=[p1,p2,p3,p4]
print p

mat=np.array([
    [1,1,1,1],
    [0,1,0,1],
    [0,0,1,1],
    [0,0,0,1]
])

print mat

matrix = np.zeros((4,4))

for x,i in enumerate(mat):
    for y,j in enumerate(i):
        if j>0:
            matrix[x][y] = p[x]*p[y]

print matrix
print matrix.sum()


[0.49, 0.38, 0.1, 0.03]
[[1 1 1 1]
 [0 1 0 1]
 [0 0 1 1]
 [0 0 0 1]]
[[ 0.2401  0.1862  0.049   0.0147]
 [ 0.      0.1444  0.      0.0114]
 [ 0.      0.      0.01    0.003 ]
 [ 0.      0.      0.      0.0009]]
0.6597


В супермаркете 60% яблок из Турции и 40% яблок из Индии. 10% турецких и 15% индийских яблок — червивые. Какова вероятность, что яблоко, купленное в этом магазине, окажется червивым? Запишите ответ от 0 до 1 с двумя знаками после десятичной точки.

In [25]:
tur=0.6
tur2=0.1

indi=0.4
indi2=0.15

x= tur2*tur
print x
y= indi2*indi
print y
xy = x+y

round(xy,2)

0.06
0.06


0.12

P(B)=PBAP(A)+PBA¯P(A¯)=0.1⋅0.6+0.15⋅0.4=0.12.

Представьте, что вы купили яблоко в магазине из предыдущей задачи, и оно оказалось червивым. Какова вероятность того, что оно из Турции, а не из Индии?

In [29]:
print x/xy

0.5



По формуле Байеса P(A|B)=P(B|A)P(A)/P(B)=0.1⋅0.6/0.12=0.5.

Журнал "Популярная механика" читает 6.4% целевой аудитории вашего продукта, а журнал "Кот Шрёдингера" — 3.7%. Если вы разместите рекламу в обоих журналах, что можно сказать о доле p целевой аудитории, которая увидит рекламу хотя бы один раз?

In [89]:
pop=0.064
kot=0.037

x= pop+kot
print x

y=x-kot
print y


0.101
0.064


P (A + B) = P (A) + P (B) − P (AB).

p=0.064, если все, кто читает pop, читают и kot  , P (AB) = P (A)

p=0.101, если аудитории этих журналов не пересекаются.

### Биномиальное,  тест  про баскетбол

In [97]:
k=1
n=10

import math


ckn= math.factorial(n)/(1*math.factorial(n-k))
print ckn

ckn*0.2*(1-0.2)**9

10


0.2684354560000001